templates/terraform/modules/environment/main.tf

```diff
@@ -52,8 +41,9 @@ data "aws_caller_identity" "current" {}
 #
 # Provision the EKS cluster
 module "eks" {
+  count = var.serverless_enabled ? 0 : 1
   source  = "commitdev/zero/aws//modules/eks"
```

Comment:
two forward slashes here... is that on purpose?

Reply:
Yes, this syntax is used to refer to a sub module. https://www.terraform.io/language/modules/sources#modules-in-package-sub-directories